### ISSUE TITLE
Cannot run oledir on 2.7.10

### ISSUE THREAD

**Issue by peres** [decalage2/oletools#204]
With python 2.7.10 on macOS and virtualenv, I cannot run _oledir_:
```
Traceback (most recent call last):
  File "/Users/XXX/.pyenv/versions/2.7.10/bin/oledir", line 9, in <module>
    load_entry_point('oletools==0.51', 'console_scripts', 'oledir')()
  File "/Users/XXX/.pyenv/versions/2.7.10/lib/python2.7/site-packages/oletools/oledir.py", line 218, in main
    left  = sid_display(d.sid_left)
  File "/Users/XXX/.pyenv/versions/2.7.10/lib/python2.7/site-packages/oletools/oledir.py", line 125, in sid_display
    if sid == olefile.NOSTREAM:
AttributeError: 'module' object has no attribute 'NOSTREAM'
```

**Comment by peres** [comment]
The problem I described above happens with the installation via pip, which I tried yesterday. Running _oledir_ from within a cloned repo works fine.

**Comment by decalage2** [comment]
This issue was fixed by commit 433f234ff872a59fb6af5c10f9c4876a8c7002d0
